src/MLOps.NET/Catalogs/EvaluationCatalog.cs

```diff
@@ -49,7 +49,10 @@ public async Task LogMetricAsync(Guid runId, string metricName, double metricVal
 
             foreach (var metric in properties)
             {
-                await LogMetricAsync(runId, metric.Name, (double)metric.GetValue(metrics));
+                if (!(double.IsNaN((double)metric.GetValue(metrics))))
```

Comment:
```suggestion
               var value = (double)metric.GetValue(metrics);
                if (double.IsNaN(metric)) continue;
```

Reply:
Should - if (double.IsNaN(metric)) continue;

not be this - if (double.IsNaN(value)) continue;

@aslotte

Reply:
Great catch, yeap!